src/patterns/help-users-contact-a-department/all-contact-information/index.njk

```diff
@@ -0,0 +1,28 @@
+---
+title: All contact information
+layout: layout-example.njk
+---
+
+<p class="govuk-body govuk-!-font-weight-bold">Get help with your application</p>
+
+<ul class="govuk-list">
+  <li>Telephone: 0800 100 1234</li>
```

Comment:
Do we know if these numbers are associated with any real organisations?

I searched for it and it seems it might be in use, referenced on this page:

https://members.scouts.org.uk/supportresources/4164/divorce-and-separation

For example Ofcom has telephone numbers for use in TV and radio drama programmes
 

https://www.ofcom.org.uk/phones-telecoms-and-internet/information-for-industry/numbering/numbers-for-drama

Reply:
I have changed the numbers to be generic Ofcom numbers for drama